### DISCUSSION TITLE
plakar: snapshot forking and tagging %C2%B7 poolp.org

### DISCUSSION THREAD

**Discussion by giscus** [poolpOrg/poolp.org#165] [Articles]
# plakar: snapshot forking and tagging %C2%B7 poolp.org

TL;DR: plakar gained two minor features, snapshots fork-ing and snapshots tagging, which makes it possible to unlock several major new features in a relatively short future.

https://poolp.org/posts/2023-07-23/plakar-snapshot-forking-and-tagging/

**Comment by fcassin** [comment]
Thanks for the article!

I was not sold on the GPT co-author, but I'm now a fan, if only for the comic relief :)

**Reply by poolpOrg** [reply]
He's still in search of a personality and style, I think he was a bit nervous about his first interventions :-)
